src/sync/installation.ts

```diff
@@ -240,38 +261,37 @@ export const processInstallation =
 
 			const processor = tasks[task];
 
-			const handleGitHubError = (err) => {
-				if (err.errors) {
-					const ignoredErrorTypes = ["MAX_NODE_LIMIT_EXCEEDED"];
-					const notIgnoredError = err.errors.filter(
-						(error) => !ignoredErrorTypes.includes(error.type)
-					).length;
-
-					if (notIgnoredError) {
-						throw err;
-					}
-				} else {
-					throw err;
-				}
-			};
-
 			const execute = async () => {
 				if (await booleanFlag(BooleanFlags.SIMPLER_PROCESSOR, true)) {
-					try {
-						return await processor(github, repository, cursor, 20);
-					} catch (err) {
-						logger.error({ err, job, github, repository, cursor, task }, "Error Executing Task");
-						handleGitHubError(err);
-					}
+
+					// just try with one page size
+					return await processor(github, repository, cursor, 20);
```

Comment:
The `SIMPLER_PROCESSOR` flag is `true` in prod, currently. I removed the `try/catch` and `handleGitHubError()` call here, because we're not iterating through different page sizes and `handleGitHubError()` would just have re-thrown the error anyways.